src/Sylius/Bundle/CoreBundle/Form/EventSubscriber/ContactFormSubscriber.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\EventSubscriber;
+
+use Sylius\Component\Customer\Context\CustomerContextInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+final class ContactFormSubscriber implements EventSubscriberInterface
```

Comment:
Let's just write it as:

```php
$builder->addEventListener(FormEvents::PRE_SET_DATA, function (FormEvent $event) {
    // preSetData body
});
```

inside ContactType.

There's no need to make publicly available subscriber to use it only in one form type.